eth/ethconfig/config.go

```diff
@@ -182,12 +182,6 @@ type Config struct {
 	// Miscellaneous options
 	DocRoot string `toml:"-"`
 
-	// Type of the EWASM interpreter ("" for default)
-	EWASMInterpreter string
-
-	// Type of the EVM interpreter ("" for default)
-	EVMInterpreter string
```

Comment:
Given that the config file will be rejected if we remove these, I think for now we should keep these and add a doc that they are deprecated/unused. We can remove them in a future PR if Felix changes the toml parser to accept invalid fields (or somehow more gracefully deprecate ones).